ci/lint/04_install.sh

```diff
@@ -9,6 +9,7 @@ export LC_ALL=C
 travis_retry pip3 install codespell==1.15.0
 travis_retry pip3 install flake8==3.7.8
 travis_retry pip3 install yq
+travis_retry pip3 install mypy
```

Comment:
Can this be set to a specific version or is this assumed to be a stable program? We had trillion of issues with flake8 and codespell being updated from down under, so I'd like to avoid any such mishaps in the future.

Reply:
Looks like there is a new release every month or so: https://github.com/python/mypy/releases

So it makes sense to set: `travis_retry pip3 install mypy==0.770`

Reply:
@MarcoFalke Done